src/components/about/About.js

```diff
@@ -114,31 +113,30 @@ const About = () => {
               </span>
             </h2>
             <p>
-              The goal of the Twitterbot on the Blue Witness project is to
-              adequately scrape twitter for reports of incidents of police
+              The goal of the Twitter bot on the Blue Witness project is to
+              adequately scrape Twitter for reports of incidents of police
```

Comment:
very nit picky, but could maybe change this sentence to something like "...reports regarding incidents of police..." just to have a bit more flow?  Just a thought, feel free to do whatever you want.

Reply:
Yeah, I agree it sounds better thanks.